### PR TITLE
Enable alternate appview proxy on pds

### PR DESCRIPTION
**Pull request by dholms**
Adds two new env variables for configuring an alternate appview url & a subset of users to use that appview

### CODE REVIEW

File: packages/pds/src/context.ts [hunk 1]
```diff
@@ -162,6 +168,16 @@ export class AppContext {
...
     const appViewAgent = new AtpAgent({ service: cfg.bskyAppView.url })
+
+    const alternateAppviewAgent = cfg.bskyAppView.alternate
+      ? new AtpAgent({ service: cfg.bskyAppView.alternate.url })
+      : undefined
+    const alternateAppviewUsers = (
+      cfg.bskyAppView.alternate?.users ?? []
+    ).reduce((acc, cur) => {
+      return acc.set(cur, true)
+    }, new Map<string, boolean>())
```
Comment (devinivy) [inline packages/pds/src/context.ts:179]: Really doesn't make a practical difference but a set would be quite tidy here!
```ts
const alternateAppviewUsers = new Set(cfg.bskyAppView.alternate?.users)
```

**Review by devinivy — APPROVED** [review]
Awesome, thanks!

File: packages/pds/src/context.ts (hunk 1 above)
Comment (dholms) [reply packages/pds/src/context.ts:179]: oh doh

**Comment by dholms** [comment]
Closing - no longer needed

**Pull request closed without merging**